pkg/util/fs_util.go

```diff
@@ -188,9 +188,11 @@ func extractFile(dest string, hdr *tar.Header, tr io.Reader) error {
 	switch hdr.Typeflag {
 	case tar.TypeReg:
 		logrus.Debugf("creating file %s", path)
-		// It's possible a file is in the tar before its directory.
-		if _, err := os.Stat(dir); os.IsNotExist(err) {
-			logrus.Debugf("base %s for file %s does not exist. Creating.", base, path)
+		// It's possible a file is in the tar before its directory,
+		// or a file was copied over a directory prior to now
+		fi, err := os.Stat(dir)
+		if os.IsNotExist(err) || !fi.IsDir() {
+			logrus.Infof("base %s for file %s does not exist. Creating.", base, path)
```

Comment:
WDYT of changing this log level back to `debugf` so that the output isn't too cluttered?

Reply:
sounds fine.